cascades-server/configuration/src/test/java/pl/gov/coi/cascades/server/presentation/PresentationConfigurationTest.java

```diff
@@ -26,11 +30,10 @@
     @Rule
     public MockitoRule mockitoRule = MockitoJUnit.rule();
 
-    @Test
-    public void testProduceTemplateIdGeneratorService() throws Exception {
-        // given
-        PresentationConfiguration presentationConfiguration = new PresentationConfiguration();
+    private final PresentationConfiguration presentationConfiguration = new PresentationConfiguration();
```

Comment:
In tests produce an instance by passing `Random` initialized with a constant seed, like that `new Random(123456L)`.

In that way you will be able to assert that produced password is exactly something like that: 'QAss23dd2sss'. It will not change until generation algorithm has, with you like to check.